py3status/modules/volume_status.py

```diff
@@ -149,6 +143,97 @@ def current_volume(self):
         # get info about channel mute status
         muted = self._get_muted(output)
 
+        return perc, muted
+
+    def volume_up(self, delta):
+        # volume up
+        cmd = 'amixer -q -D {} sset {} '.format(self.device, self.channel)
```

Comment:
fine but can we do the `call()` in one place this file now has `DEVNULL = open(devnull, 'wb')` seven times which feels overkill to me if you are going to have a base class at least use it for something